packages/core/core/src/utils/template_generator.js

```diff
@@ -46,7 +51,7 @@ class TemplateGenerator {
   async download(url, tmpFilePath, browse) {
     console.log(__('Installing template from ' + browse).green);
     console.log(__('Downloading template...').green);
-    fs.mkdirpSync(utils.dirname(tmpFilePath));
+    fs.mkdirpSync(path.dirname(tmpFilePath));
```

Comment:
Turns out `utils.dirname()` didn't add any value over `path.dirname()` so I got rid off it as well

Reply:
I think it was part of abstracting `path`, but the value it's not worth it